go/pkg/marketplace/marketplace_service.go

```diff
@@ -73,11 +74,18 @@ type DBCollectionWithExtra struct {
 	Name                string
 	ImageURI            string
 	Volume              string
+	TotalVolume         float32
+	FloorPrice          uint64
+	MintPrice           string
+	NumTrades           int64
+	NumOwners           int32
+	Denom               string
 	MintContractAddress string
 	CreatorAddress      string
-	Price               uint64
+	Price               string
```

Comment:
Should double-check if this Price is used somewhere for calculation (front/back), and the change from uint64 => string does not break that calculation

Reply:
I've checked and I didn't find any issues.